app/pods/application/adapter.js

```diff
@@ -4,5 +4,5 @@ import DS from 'ember-data';
 import config from '../../config/environment';
 
 export default DS.FirebaseAdapter.extend({
-  firebase: new Firebase(`https://${config.firebase_instance}.firebaseIO.com`)
+  firebase: new Firebase(`https://${config.firebaseUrl}.firebaseIO.com`)
```

Comment:
Perhaps would be a little more clear to just have the whole url in the config variable?